orderbook/openapi.yml

```diff
@@ -265,6 +291,29 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Trade:
+      description: |
+        Trade data such as executed amounts, fees, order id and block number.
+      type: object
+      properties:
+        blockNumber:
+          description: "Block in which trade occurred"
+          type: integer
+        logIndex:
+          description: "Index in which transaction was included in block"
+          type: integer
+        orderUid:
```

Comment:
why orderUid and not order Id? 
people would expect ids to be unique anyways....
normally u see just id everywhere

Reply:
orderUid is taken from the smart contracts. If it's only for the unnecessary u in the name, I'd prefer to stick to the smart contract notation (less ambiguity what this can be used for).

Reply:
Note also that in the `Orders` table this is `UID` as the Primary Key and in the `Trades` as `orderUid` as a Foreign Key (that is, assuming my understanding of PKs and FKs are correct).

Reply:
okokok, never mind, yes I don't like that U, but I can live with it. Scratch this

Reply:
Me neither, but it stays.

Reply:
There is an additional train of thought that supports `Uid` here and that is the fact that an individual user might want to see their order ID as a sequence of integers (i.e. first order, second order, etc..) while these, user specific, order IDs are not universally unique over all, they are still IDs from the owners perspective. Hence the distinction between Unique ID and ID

Reply:
I'm not sure I got this point
> while these, user specific, order IDs are not universally unique over all

You say "Universally" because we considered using UUID, but we went for a unique ID that is not universal and we want to specify this kind of things in it's name. 

Fine with me, as I say, it feels strange to me, I never see people using this naming convention. UUIDs are a special data type with it's RFC spec. Anyways, is not that important to me, I just wanted to mention it